files/build_templates/sonic_debian_extension.j2

```diff
@@ -40,6 +40,7 @@ FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMPLATES="$FILESYSTEM_ROOT_USR_SHARE_SONIC/temp
 FILESYSTEM_ROOT_ETC="$FILESYSTEM_ROOT/etc"
 FILESYSTEM_ROOT_ETC_SONIC="$FILESYSTEM_ROOT_ETC/sonic"
 
+
```

Comment:
Delete this extra blank line.

Reply:
@jleveque Just did it before you comment.